src/app/Actions.js

```diff
@@ -1,53 +1,76 @@
+    CLEAR_OFFLINE_MATCHES: '[CACHE] Clear matches',
+    SUBMIT_AUTO_NOTES: '[QUAL] Submit Auto Notes',
+    SUBMIT_COLLECTION_NOTES: '[QUAL] Submit Collection Notes',
+    SUBMIT_SHOOT_POS_NOTES: '[QUAL] Submiit Shooting Position Notes',
+    SUBMIT_SHOOT_COS_NOTES: '[QUAL] Submit Shooting Consistency Notes',
+    SUBMIT_PATH_NOTES: '[QUAL] Submit Path Notes',
+    SUBMIT_DEFENSE_NOTES: '[QUAL] Submit Defense Notes',
+    SUBMIT_CLIMBING_NOTES: '[QUAL] Submit Climing Notes',
+    SUBMIT_HP_NOTES: '[QUAL] Submit Human Player Notes',
+    SUBMIT_PENALTY_NOTES: '[QUAL] Submit Penalties Notes',
+    SUBMIT_DRIVERS_NOTES: '[QUAL] Submit Drivers Notes'
 };
 
 
 export const resetState = () => {
-	return {
-		type: ActionTypes.RESET_STATE
-	};
+    return {
+        type: ActionTypes.RESET_STATE
+    };
 };
 
 export const getOfflineMatchesSuccess = (matches) => {
-	return {
-		type: ActionTypes.GET_OFFLINE_MATCHES_SUCCESS,
-		payload: matches
-	};
+    return {
+        type: ActionTypes.GET_OFFLINE_MATCHES_SUCCESS,
+        payload: matches
+    };
 };
-
 export const clearOfflineMatches = () => {
-	return {
-		type: ActionTypes.CLEAR_OFFLINE_MATCHES,
-	};
-};
-
-export const activateNode = (gamemode, index) => {
-	const type = (gamemode === 'teleop') ? ActionTypes.ACTIVATE_TELEOP_NODE : ActionTypes.ACTIVATE_AUTO_NODE;
-	return {
-		type: type,
-		payload: index
-	};
-};
-
-export const setAutoPark = (value) => {
-	return {
-		type: ActionTypes.SET_AUTO_PARK,
-		payload: value
-	};
-};
-
-export const setChargeStation = (gamemode, value) => {
-	return {
-		type: (gamemode === 'teleop') ? ActionTypes.SET_TELEOP_CHARGE_STATION : ActionTypes.SET_AUTO_CHARGE_STATION,
-		payload: value
-	};
+    return {
+        type: ActionTypes.CLEAR_OFFLINE_MATCHES,
+    };
 };
+export const sendNotes = (topic, content) => {
+    let type;
+    switch (topic) {
+        case 'Auto':
+            type = ActionTypes.SUBMIT_AUTO_NOTES
+            break;
+        case 'Collection':
+            type = ActionTypes.SUBMIT_COLLECTION_NOTES
+            break;
+        case 'Shooting Position':
+            type = ActionTypes.SUBMIT_SHOOT_POS_NOTES
+            break;
+        case 'Shooting Consistency':
+            type = ActionTypes.SUBMIT_SHOOT_COS_NOTES
+            break;
+        case 'Path':
+            type = ActionTypes.SUBMIT_PATH_NOTES
+            break;
+        case 'Defense':
+            type = ActionTypes.SUBMIT_DEFENSE_NOTES
+            break;
+        case 'Climbing':
+            type = ActionTypes.SUBMIT_CLIMBING_NOTES
+            break;
+        case 'Human Player':
+            type = ActionTypes.SUBMIT_HP_NOTES
+            break;
+        case 'Penalties':
+            type = ActionTypes.SUBMIT_PENALTY_NOTES
+            break;
+        case 'Drivers':
+            type = ActionTypes.SUBMIT_DRIVERS_NOTES
+            break;
+        default:
+            
+    }
+    return {
+    type,
+    payload: content, topic
```

Comment:
You can't just comma-separate these values. This is the equivalent of saying
```
return {
  type: type,
  payload: content,
  topic: topic
};
```
But `type` and `payload` are the only two properties that should exist on an Action object. So you should remove `, topic` entirely.